internal/transport/transport.go

```diff
@@ -453,6 +453,7 @@ func webWorker(ctx context.Context, workerID int, jobs <-chan *webJob) {
 			WorkerID:   workerID,
 			WorkerName: "web",
 			Duration:   time.Since(start),
+			Host:       rsp.Request.URL.Host,
```

Comment:
We should escape the host before logging it. See [here](https://github.com/alpine-hodler/gidari/blob/main/internal/transport/transport.go#L449-L450) on how to do that.